packages/playwright/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "@sap-ux-private/playwright",
```

Comment:
Yes, currently there is no such request and changing it to public should be easy once there is such request. I also do not have any objection to make it public now, but [admin](https://github.com/orgs/SAP/teams/ux-tools-admin) should agree

Reply:
@SAP/ux-tools-admin Any objections to make this public?

Reply:
I am not opposed to making it public. 
But private is ok for now it we don't have concrete plans to consume back in tools-suite. It can be changed later.